blurb/README.rst

```diff
@@ -247,6 +247,13 @@ part of the cherry-picking process.
 Changelog
 ---------
 
+1.0.8
+~~~~~
+
+- Added the ``--bpo`` and ``--section`` flags to
```

Comment:
Does it make sense to use the bpo abbreviation, or would it be better as `issue` now?